src/github.com/getlantern/flashlight/feed/feed.go

```diff
@@ -297,3 +289,20 @@ func determineLocale(defaultLocale string) string {
 	}
 	return defaultLocale
 }
+
+func getHTTPClient() (*http.Client, error) {
+	var err error
+	httpClientMutex.Lock()
+	if _httpClient == nil {
+		_httpClient, err = proxied.ChainedNonPersistent("")
```

Comment:
Cool. I don't love the laziness aspect of this in the sense that there's nothing we're really saving (creating the HTTPClient is super cheap), but the fact the ChainedNonPersistent can return an error makes a solution unclear. To me it would be significantly nicer if we could just say `_httpClient     = getHTTPClient()` at the beginning and skip all of this, but that would require something like calling `getProxyAddr` once at startup in `proxied.go` and then just always using that, failing fast potentially just at startup if that ever fails. That should never fail, though, since it's just the proxy listening at 127.